modules/mobile/spec/request/observations_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/mr_client_helpers'
+require 'medical_records/client'
+require_relative '../support/helpers/sis_session_helper'
+
+RSpec.describe 'observations', type: :request do
+  include MedicalRecords::ClientHelpers
+  include SchemaMatchers
+
+  let!(:user) { sis_user(icn: '32000225') }
+
+  let(:mhv_account_type) { 'Premium' }
+
+  let(:observation_response) do
+    {
+      'id' => 'I2-ILWORI4YUOUAR5H2GCH6ATEFRM000000',
+      'type' => 'observation',
+      'attributes' => {
+        'status' => 'final',
+        'category' => [
+          {
+            'coding' => [
+              {
+                'system' => 'http://terminology.hl7.org/CodeSystem/observation-category',
+                'code' => 'laboratory',
+                'display' => 'Laboratory'
+              }
+            ],
+            'text' => 'Laboratory'
+          }
+        ],
+        'code' => {
+          'coding' => [
+            {
+              'system' => 'http://loinc.org',
+              'code' => '2339-0',
+              'display' => 'Glucose'
+            }
+          ],
+          'text' => 'Glucose'
+        },
+        'subject' => {
+          'reference' => 'https://sandbox-api.va.gov/services/fhir/v0/r4/Patient/1000005',
+          'display' => 'Mr. Shane235 Bartell116'
+        },
+        'effectiveDateTime' => '1998-03-16T05:56:37Z',
+        'issued' => '1998-03-16T05:56:37Z',
+        'performer' => [
+          {
+            'reference' => 'https://sandbox-api.va.gov/services/fhir/v0/r4/Practitioner/I2-4ZXYC2SQAZCHMOWPPFNLOY65GE000000',
+            'display' => 'DR. THOMAS359 REYNOLDS206 PHD'
+          }
+        ],
+        'valueQuantity' => {
+          'value' => 78.278855002875,
+          'unit' => 'mg/dL',
+          'system' => 'http://unitsofmeasure.org',
+          'code' => 'mg/dL'
+        }
+      }
+    }
+  end
+
+  before do
+    allow(MedicalRecords::Client).to receive(:new).and_return(authenticated_client)
```

Comment:
It looks like you copied this test from somewhere in my_health, like perhaps the allergies request spec. One thing I find confusing is that the authenticated client returns a client with a totally different user id. The `spec/support/vcr_cassettes/rrd/lighthouse_observation.yml` cassette actually uses the user id you specified above:
```
        X-Va-Includes-Icn:
          - '32000225'
```
Also, the authenticated client is specific to `MedicalRecords::Client` while our controller uses `Lighthouse::VeteransHealth::Client`. I haven't followed the thread all the way through, but it's not clear to me that `MedicalRecords::Client` is used here at all.

I removed this line and the tests still pass, so I'm not convinced it's necessary.

Reply:
Sure, I can remove it